### DISCUSSION TITLE
creating a round burner

### DISCUSSION THREAD

**Discussion by Dukeswp** [firemodels/fds#12689] [Q&A]
hello guys, as a novice using FDS, I want to know how to create a round burner in Pyrosim. I have tried to create the round burner by rotating obstacles, but it does not look good. And second, If I choose to create a circular vent from the vent menu, the burner does not burn.  Meanwhile, I tried to create one by CAD. However, I found that I was unable to select one side (the top interface) of the model as a burning interface. I do really want to know how you there build a satisfying round burner?
Thanks!

**Comment by rmcdermo** [comment]
Have a look at the FDS user guide sections on MULT and Complex Geometry (GEOM).
